apps/service-portal/src/components/Layout/Layout.css.ts

```diff
@@ -7,7 +7,8 @@ export const layoutWrapper = style({
 })
 
 export const layoutGrid = style({
-  transition: 'all 150ms ease-in-out',
+  transition: 'margin 150ms ease-in-out, flexBasis 150ms ease-in-out',
+  willChange: 'transition',
```

Comment:
Shouldn't this be `margin, flexBasis` ... since you're transitioning the margin + flexBasis.

Reply:
whops - I've never seen this before 😄 fixed